test/e2e/tests/confirmations/transactions/erc721-revoke-set-approval-for-all-redesign.ts

```diff
@@ -44,30 +44,6 @@ async function mocks(server: Mockttp) {
   return [await mocked4BytesSetApprovalForAll(server)];
 }
 
-export async function mocked4BytesSetApprovalForAll(mockServer: Mockttp) {
```

Comment:
no exports in spec files. Moved to helpers files, as it's used in more than one test